### ISSUE TITLE
Questions regarding the data for fine-tuning the operational version of GraphCast

### ISSUE THREAD

**Issue by LQscience** [google-deepmind/graphcast#190]
The model is described as being `
'GraphCast_operational, a high-resolution model (0.25 degree resolution, 13 pressure levels) pre-trained on ERA5 data from 1979 to 2017 and fine-tuned on HRES data from 2016 to 2021.'
`

In the fine-tuning phase, what dataset is used as the target data? Is loss calculated between GraphCast outputs and ERA5 data? 
Or precipitation of ERA5 data but other variables of HRES-fc0?
